multi_map_it_test.go

```diff
@@ -450,3 +450,120 @@ func TestMultiMap_TryLockWithLeaseAndTimeout(t *testing.T) {
 		assert.False(t, b)
 	})
 }
+
+func TestMultiMap_ContainsKey(t *testing.T) {
+	it.MultiMapTester(t, func(t *testing.T, m *hz.MultiMap) {
+		ctx := context.Background()
+		keyExists, err := m.ContainsKey(ctx, "key")
```

Comment:
How about `exists` instead of `keyExists`?

Reply:
Thanks, fixed it at [`c1204cb`](https://github.com/hazelcast/hazelcast-go-client/pull/697/commits/c1204cbf69648056914f16f60ce67946a92543bd)